lms/envs/devstack.py

```diff
@@ -230,3 +230,6 @@ def should_show_debug_toolbar(_):
 MODULESTORE = convert_module_store_setting_if_needed(MODULESTORE)
 
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
+
+# For the metrics tab.
+LMS_BASE = 'localhost:8000'
```

Comment:
Odd; this is currently set and used by CMS and _technically_ referenced by LMS, but in a very hacky way (with a note that says `# TODO: bbeggs remove this before prod, temp fix to get load testing running`).
As is, though, this value is never defined in either `aws.py` nor `common.py`, so I would expect this to break in production.

Should we instead set this to localhost in `common.py` and then pull it from json in `aws.py`?
It looks like configuration is already sticking it into the json.